### PR TITLE
New package: spek-alternative-0.8.2.3

### PR DESCRIPTION
**Pull request by kartikynwa**

### CODE REVIEW

File: srcpkgs/spek-alternative/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'spek-alternative'
+pkgname=spek-alternative
+version=0.8.2.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake gettext-devel intltool libtool pkg-config wxWidgets-common"
```
Comment (abenson) [inline srcpkgs/spek-alternative/template:6]: What does it need from `wxWidgets-common` in `hostmakedepends`?

Comment (kartikynwa) [reply srcpkgs/spek-alternative/template:6]: Yep

File: srcpkgs/spek-alternative/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+makedepends="ffmpeg-devel wxWidgets-devel"
```
Comment (abenson) [inline srcpkgs/spek-alternative/template:7]: I think should be `wxWidgets-gtk3-devel`

Comment (kartikynwa) [reply srcpkgs/spek-alternative/template:7]: I'll try building and get back to you

Comment (kartikynwa) [reply srcpkgs/spek-alternative/template:7]: It fails with this output: http://dpaste.com/3HKXJWW.txt

This is the template I used: http://dpaste.com/370Q620.txt

Comment (abenson) [reply srcpkgs/spek-alternative/template:7]: Yeah, the gtk3 version uses `wx-config-gtk3`.

Comment (kartikynwa) [reply srcpkgs/spek-alternative/template:7]: I am sorry. Will try again and let you know.

Comment (ericonr) [reply srcpkgs/spek-alternative/template:7]: wxWidgets-gtk3-devel has the `wx-config-gtk3` binary. You might need to tell the build system to use it, if it doesn't use the `.pc` correctly. But it should be fine.

Comment (ericonr) [reply srcpkgs/spek-alternative/template:7]: It's defined here but I can't understand what it's doing. https://github.com/withmorten/spek-alternative/blob/0.8.2.3/configure.ac#L53

You might try opening an issue with the dev asking how to specify the wx-config binary.

Comment (kartikynwa) [reply srcpkgs/spek-alternative/template:7]: Somehow I understand. I have opened an issue.

Seems like the AM_OPTIONS_WXCONFIG is defined in `/usr/share/aclocal/wxwin.m4` in package `wxWidgets-common`. I'll look into it tomorrow more and see if I can spin something up.

Comment (kartikynwa) [reply srcpkgs/spek-alternative/template]: That got taken care of. I'll mark this as resolved. Thanks for helping me out.

Comment (ericonr) [reply srcpkgs/spek-alternative/template]: well that works, great!

**Comment by ericonr** [comment]
@kartikynwa might make sense to ask the dev to make a new release, there seem to have been quite a few fixes accumulated after the last release.

**Pull request merged**